examples/experimental/export_example.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    # Install zipkin python package:
</pre><p>Possible to show this with the standard opentelemetry collector instead of zipkin? <a href="https://opentelemetry.io/docs/collector/quick-start" target="_blank">https://opentelemetry.io/docs/collector/quick-start/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1346/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
You mean the toy collector that merely outputs the spans to a text file?

Reply:
Yeah, maybe in addition? Want to show more clearly what the spans look like.

Unrelatedly, how easy is it for someone to choose a different exporter? Should we show this for 2-3 of the most popular ones?